tests/phpunit/tests/test-admin-filters.php

```diff
@@ -15,38 +15,42 @@ function setUp() {
 
 		self::$polylang->filters = new PLL_Admin_Filters( self::$polylang );
 	}
-
-	function test_sanitize_title() {
+	function test_sanitize_title_for_current_language_without_character_conversion() {
 		$pll_admin = new PLL_Admin( self::$polylang->links_model );
 		$pll_admin->curlang = self::$polylang->model->get_language( 'en' );
 		$pll_admin->add_filters();
 		$this->assertEquals( 'fullmenge', sanitize_title( 'Füllmenge' ) );
-
+	}
+	function test_sanitize_title_for_language_from_form_without_character_conversion() {
+			// Bug fixed in 2.4.1
+		$_POST['post_lang_choice'] = 'en';
+		$pll_admin = new PLL_Admin( self::$polylang->links_model );
+		$pll_admin->add_filters();
+		$this->assertEquals( 'fullmenge', sanitize_title( 'Füllmenge' ) );
+	}
```

Comment:
These tests always pass, whether we set the filters or not.

Reply:
🤔 These tests exist before. I think that this the goal: verify that our filter doesn't break anything for standard languages - just removing accents.
Notice that WordPress apply a default sanitization by applying sanitize_title filter and then executing sanitize_title_with_dashes() formating function 
https://developer.wordpress.org/reference/functions/sanitize_title_with_dashes/